### ISSUE TITLE
Harvest LandOwnership from SITLA AGOL Item back to SGID internal

### ISSUE THREAD

**Issue by stdavis** [agrc/warehouse#114]
IIRC this is done via script by Barry currently?

@gregbunce said that SITLA's AGOL items are currently backing our open data items. Do we want to replace Barry's update process with a pallet that gets data directly from their AGOL Item?

**Comment by gregbunce** [comment]
i like that idea.  but, i'll have to reach out to him and see what think.  also, I've asked if they'd be open to change the name of their service from "[UT SITLA Ownership LandOwnership WM](https://opendata.gis.utah.gov/datasets/SITLA::ut-sitla-ownership-landownership-wm-1?geometry=-129.191%2C36.998%2C-93.904%2C42.002)" to "Utah Land Ownership" for something similar.

**Comment by gregbunce** [comment]
Barry is fine with us updating our internal land ownership layer via their agol.  will the current name they are using be an issue?

**Comment by stdavis** [comment]
No, I don't believe that the title would have any effect on this process.
